spec/controllers/faye_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe FayeController do
+  include Devise::TestHelpers
+  before :each do
+    user = create :user, admin: true
```

Comment:
It seems that when I use the when_current_user_is :admin method the test fails. However, when I leave it as is the test passes?